engine/src/execution_graph/logic_controllers/LogicalFilter.cpp

```diff
@@ -125,6 +125,11 @@ std::unique_ptr<ral::frame::BlazingTable> process_filter(
       }
     }
     assert(found_self_partition);
+
+		if( ral::utilities::checkIfConcatenatingStringsWillOverflow(partitions_to_concat)) {
+			std::cout<<"WARNING: In process_distribution_table Concatenating Strings will overflow strings length"<<std::endl;
```

Comment:
Have this std::cout be output to the logs. 
This function is called by bc.partition, which does not have a query id or anything like that, so you can leave all the logging fields blank, encept for the `info` where you provide this warning.
Also, since you are here, can you add a comment at the declaration of this function that this is only used by `bc.partition`